stdlib/2and3/_curses.pyi

```diff
@@ -355,9 +355,9 @@ class _CursesWindow:
     def getch(self, y: int, x: int) -> _chtype: ...
     if sys.version_info >= (3, 3):
         @overload
-        def get_wch(self) -> _chtype: ...
+        def get_wch(self) -> Union[int, str]: ...
         @overload
-        def get_wch(self, y: int, x: int) -> _chtype: ...
+        def get_wch(self, y: int, x: int) -> Union[int, str]: ...
```

Comment:
These should return `Any` per our guidelines. Returning `Unions` is inconvenient for callers, since it requires `isinstance` checks. (Even if this used a `Union` previously.)

Reply:
hmmm, I think it's pretty important to return the union type here as the two types can't really be considered the same.  I read through the rationale "the caller needs to use `isinstance`" but that's true here already because `int` and `str` have very little interface overlap

it's also not a regression, this was already returning a union and I'm simply correcting the union

Reply:
Ok.